webpay/settings/base.py

```diff
@@ -27,6 +27,8 @@
 DEBUG = True
 DEBUG_PROPAGATE_EXCEPTIONS = True
 
+USE_TZ = True
```

Comment:
Maybe a brief comment could be added to explain what this does or link to the docs if that's easier.

What does this change in context of the running system mean - e.g is all the datetime handling ok? Will this impact anything else webpay talks to?